src/Evolution/Systems/CurvedScalarWave/Worldtube/Tags.hpp

```diff
@@ -70,6 +70,17 @@ struct SelfForceOptions {
   using group = Worldtube;
 };
 
+/*!
+ * \brief The mass of the scalar particle
+ */
+struct Mass {
+  using type = double;
+  static constexpr Options::String help{"The mass of the scalar particle."};
```

Comment:
What are the units of this mass? Is it really in terms of M of the BH?

Reply:
Yes, I added a comment